### PR TITLE
Purchases: Add upgrade button

### PR DESCRIPTION
**Pull request by fditrapani**
#### Changes proposed in this Pull Request

This PR adds an upgrade button, and some tracking events, to the plan settings screen.

**Before**
image

**After**
image

#### Testing instructions

- Pick a plan and manage your subscription. Confirm there is an upgrade button and that it takes you to the plans, and fires an event, when you click on it.
- Manage a subscription for an eCommerce plan and confirm there is no upgrade button.
- Pick a domain or any non-plan product and confirm that there's no upgrade button.

### CODE REVIEW

File: client/me/purchases/manage-purchase/index.jsx [hunk 1]
```diff
@@ -237,6 +238,37 @@ class ManagePurchase extends Component {
...
+			status: isExpired( purchase ) ? 'expired' : 'active',
+			plan: purchase.productName,
+		} );
+	};
+
+	renderUpgradeNavItem() {
+		const { purchase, translate, siteId } = this.props;
+		const buttonText = isExpired( purchase )
+			? translate( 'Pick another plan' )
+			: translate( 'Upgrade plan' );
```
Comment (a8ci18n) [inline client/me/purchases/manage-purchase/index.jsx:254]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **36** times:
`translate( 'Upgrade Plan' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/upgrade-plan-button)

**Review by sirbrillig — APPROVED** [review]
Testing works as described!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5100423

Thank you @fditrapani for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5100423).

**Pull request merged**